liveclasses/jass_jwt.py

```
@@ -0,0 +1,138 @@
+        return self
+
+    def with_user_avatar(self, avatar_url):
+        """Returns the JaaSJwtBuilder with the avatar claim set."""
+
+        self.user_claims['avatar'] = avatar_url
+        return self
+
+    def with_moderator(self, is_moderator):
+        """Returns the JaaSJwtBuilder with the moderator claim set."""
+
+        self.user_claims['moderator'] = 'true' if is_moderator == True else 'false'
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if is_moderator:` for truth checks

Replace with `is_moderator`

(E712)